### PR TITLE
Add new LXC: MySpeed

### PR DESCRIPTION
**Pull request by MickLesk**
## Description
- New LXC for MySpeed (Speed Testing)
- Tested and works fine

image

Software: 
https://github.com/gnmyt/myspeed
MySpeed is a speed test analysis software that records your internet speed for up to 30 days.
MySpeed generates clear statistics on speed, ping, and more
MySpeed automates speed tests and allows you to set the time between tests using Cron expressions
Add multiple servers directly to a MySpeed instance
Configure health checks to notify you via email, Signal, WhatsApp, or Telegram in case of errors or downtime
Test results can be stored for up to 30 days
Support for Prometheus and Grafana

## Type of change

- [x] New Script (Develop a new script or set of scripts that are fully functional and thoroughly tested)
- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.
- [x] This change requires a documentation update

### CODE REVIEW

File: ct/myspeed.sh [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+header_info
+if [[ ! -d /opt/myspeed ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+if (( $(df /boot | awk 'NR==2{gsub("%","",$5); print $5}') > 80 )); then
+  read -r -p "Warning: Storage is dangerously low, continue anyway? <y/N> " prompt
+  [[ ${prompt,,} =~ ^(y|yes)$ ]] || exit
+fi
+RELEASE=$(wget -q https://github.com/gnmyt/myspeed/releases/latest -O - | grep "title>Release" | cut -d " " -f 5)
+if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+
+msg_info "Stopping ${APP} Service"
```
Comment (tteck) [inline ct/myspeed.sh]: Forgot indention

File: ct/myspeed.sh [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  NS=""
+  MAC=""
+  VLAN=""
+  SSH="no"
+  VERB="no"
+  echo_default
+}
+
+function update_script() {
+	header_info
```
Comment (tteck) [inline ct/myspeed.sh]: Indented too much! lol

Comment (MickLesk) [reply ct/myspeed.sh]: lol, damn Visual Studio code, sometimes it makes big tabs in shell scripts

File: ct/myspeed.sh [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  header_info
+  if [[ ! -d /opt/myspeed ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+  if (( $(df /boot | awk 'NR==2{gsub("%","",$5); print $5}') > 80 )); then
+    read -r -p "Warning: Storage is dangerously low, continue anyway? <y/N> " prompt
+    [[ ${prompt,,} =~ ^(y|yes)$ ]] || exit
+  fi
+  RELEASE=$(wget -q https://github.com/gnmyt/myspeed/releases/latest -O - | grep "title>Release" | cut -d " " -f 5)
+  if [[ ! -f /opt/${APP}_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/${APP}_version.txt)" ]]; then
+
+  msg_info "Stopping ${APP} Service"
```
Comment (tteck) [inline ct/myspeed.sh:67]: Again, wrong indention. Can you not look at Spoolman that has the correct indention?

Comment (MickLesk) [reply ct/myspeed.sh:67]: Wtf. okay i edit at live in GitHub. VSCode fucked up. Give me 10 minutes

**Pull request merged**